FuzzyAutocomplete/DVTTextCompletionSession+FuzzyAutocomplete.m

```diff
@@ -22,6 +22,8 @@
 #import "FAItemScoringMethod.h"
 #import <objc/runtime.h>
 
+#define dispatch_on_main($block) (dispatch_get_current_queue() == dispatch_get_main_queue() ? $block() : dispatch_sync(dispatch_get_main_queue(), $block))
```

Comment:
We don't have any `dispatch_sync` calling `_fa_performFuzzyFiltering` from a background thread which is the only case where it would cause a deadlock, but I can work around it so we don't call `dispatch_get_current_queue()` cause it's deprecated